src/screens/PushTx.js

```diff
@@ -8,6 +8,9 @@
 import React, { useRef, useState } from 'react';
 import TxTextInput from '../components/tx/TxTextInput';
 import txApi from '../api/txApi';
+import { useNewUiEnabled } from '../hooks';
```

Comment:
These changes are also part of the other PR

Reply:
PR merged with master. Waiting Review.🙂